### DISCUSSION TITLE
save image of multi-ranger deck reads

### DISCUSSION THREAD

**Discussion by ogzsullu** [bitcraze/discussions#90] [Q&A]
Hi,
I am trying to use the multiranger_pointcloud.py example, which runs great. However, I could not manage to save the image it just created. I did some research and found this:

https://stackoverflow.com/questions/28732937/how-to-export-vispy-result-as-png-image

However, I still not be able to imply this algorithm into disconnected method of multiranger_pointcloud.py script.

I'd be glad if someone help me get the data as an image format.
Thanks

**Comment by krichardsson** [comment]
I don't know how to do it with Vispy but the easiest way would be a screen grab :-)

**Comment by ogzsullu** [comment]
I already did it :) it works fine but I want it more professional because I
have another robot system that uses pointcloud map came from drone. I have
to make it fully autonomous.

Kristoffer Richardsson ***@***.***>, 13 Haz 2022 Pzt, 15:01
tarihinde şunu yazdı:

**Reply by knmcguire** [reply]
Just a question about what you want to do: Are you sure that don't you want to convert it to a ROS point cloud format instead if you want to use the actual points? Instead of an actual image?
